int32 white_balance
---
int32 error